components/common/listingCart/ListingCartItem.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div class="py-2 px-6">
+    <div class="py-2 listing-border" />
+    <div class="is-flex is-justify-content-space-between">
+      <div class="is-flex">
+        <div>
+          <BasicImage
+            :src="avatar"
+            :alt="nft?.name"
+            class="border image is-48x48" />
+        </div>
+
+        <div
+          class="is-flex is-flex-direction-column is-justify-content-space-between ml-4 limit-width">
+          <div
+            class="has-text-weight-bold has-text-color line-height-1 no-wrap is-clipped is-ellipsis">
+            {{ nft.name }}
+          </div>
+          <div class="line-height-1 no-wrap is-clipped is-ellipsis">
+            {{ nft.collection?.name || nft.collection.id }}
+          </div>
+        </div>
+      </div>
+
+      <NeoButton
+        class="has-text-grey pt-4"
+        variant="text"
+        no-shadow
+        icon="trash"
+        icon-pack="far"
+        @click.native="listingCartStore.removeItem(nft.id)" />
+    </div>
+    <div class="is-flex is-justify-content-space-between pt-2">
+      <div class="is-flex is-flex-direction-column">
+        <div class="has-text-grey"><small>Floor</small></div>
+        <span>{{ floor }}</span>
```

Comment:
1.missing translation for "Floor"
2. replace `<small>` with `is-size-7` please

https://bulma.io/documentation/helpers/typography-helpers/

Reply:
Thanks for the link @daiagi, I believe we should include this to the style_guide.
I just recently, discovered that we were using bulma. 
Also is there a documentation of the existing components and styles so we can re-use it more efficiently?

Reply:
feel free to open an issue about the additions / changes you would like to see in the style guide